src/routes/health/index.ts

```diff
@@ -0,0 +1,33 @@
+import { Request } from 'express';
```

Comment:
I think this is good. Eventually we will want to create health endpoints for each of our containers that we can loop through, check and set the corresponding endpoint availability. As of right now, the endpoint health is set at the beginning and the end of the request, which works great for the long running `init/` endpoint. For `search/` and `filter/`, we should check if the elasticsearch and mongodb containers are running and that will more accurately represent the health of our endpoints.